### ISSUE TITLE
Unable to connect with tsh ssh to cluster node

### ISSUE THREAD

**Issue by alcor146** [gravitational/teleport#9489]
### Description

**What happened**:
I am using teleport 7.1.0 trying to connect to an ubuntu VM that is part of the teleport cluster
I m using this command:

tsh ssh --user=admin --proxy=domain.com:3080 --debug ubuntu@UUID-of-node

And it returns object not found.

I am getting this in debug mode 
[KEY AGENT] Unable to connect to SSH agent on socket: "". client/api.go:2780

It is the same for trying to login to the cluster from the vm with tsh login even after i deleted the .tsh folder

### Reproduction Steps
As minimally and precisely as possible, describe step-by-step how to reproduce the problem.
1. use teleport 7.1.0
2. try to use tsh login or tsh ssh

EDIT: It is the same with teleport 8.x.x

**Comment by programmerq** [comment]
Can you show the output of `tsh ls` after you log in to your cluster?

Double check that your user has permission to access the node in question. Where are you grabbing the uuid when you use it? Does it work if you specify the node name instead of the uuid?

`tsh ls -f json` will show both the "node name" as seen in `tsh ls`, as well as the uuid. This command codifies how I'm intending to refer to the "uuid" vs "node name"/hostname vernacular: tsh ls -f json | jq '.[] | {"uuid": .metadata.name, "node_name": .spec.hostname}'

**Comment by alcor146** [comment]
1) I can't show "tsh ls" since the problem is I can t log into the cluster or use ssh from the cluster to a the vm
2)I'am not grabbing the uuid yet, I still try to connect to the cluster
3)it doesn t work with the name, I still get "object not found"

TL;DR 
Any attempt to use tsh ssh, tsh login, tsh ls will fail with object not found
But I can access the vm in the UI of teleport

**Comment by alcor146** [comment]
The problem was that I didn't have the ssh enable in my yaml file
